templates/vue/src/components/Lightbox/media/VideoMedia/H5PVideoMedia/index.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="container">
+    <h5p-iframe
```

Comment:
Can we merge H5PIframe component into this component? I don't see a reason to have 2 separate components when this component is so minimal and the H5PIframe component is not used by any other components.

Reply:
Merged the two components together. It seems that the H5PVideo media never $emits("complete") to be caught by VideoMedia.vue.